### DISCUSSION TITLE
<title> Hide activity view for a git repo.

### DISCUSSION THREAD

**Discussion by RonakJain2** [community/community#78932] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How would I go about hiding or disabling the activity view for a github repo? Or if there is a method to edit and keep only things which you want people to see on the activity view for a github repo.

**Comment by Kiran1689** [comment]
You can't hide the activity manually.. 
If the repo is set to private, contributions to that repo will not be shown to public..
